src/api/creatures.py

```diff
@@ -46,18 +45,18 @@ def get_creatures(user_id: int):
                         ORDER BY creatures.name
                         """
                     ),
-                    {"user_id": user_id},
+                    {"user_id": user.id},
                 )
                 .mappings()
                 .all()
             )
 
         if not creatures:
             raise HTTPException(
-                status_code=404, detail=f"User {user_id} does not exist."
+                status_code=404, detail=f"User {user.id} does not exist."
```

Comment:
exceptions like these aren't needed anymore since auth will catch invalid credentials before anything else

Reply:
auth cannot catch an instance where the given user has valid credentials, but for some reason does not exist in users (either because they were deleted or never added). That is what these error messages now catch.